Model/Config.php

```diff
@@ -13,6 +13,7 @@ class Config
     public const XML_PATH_COOKIEBOT_ID = 'web/cookiebot/id';
     public const XML_PATH_DATA_CULTURE = 'web/cookiebot/data_culture';
     public const XML_PATH_USE_EU_CDN = 'web/cookiebot/use_eu_cdn';
+    public const XML_PATH_GOOGLE_CONSENT = 'web/cookiebot/google_consent';
```

Comment:
Let's rename this setting to `use_google_consent_mode`. The constant name should be names accordigly.